.rubocop.yml

```diff
@@ -108,7 +109,7 @@ Style/SpaceInsideHashLiteralBraces:
 Style/StringLiterals:
   Severity: fatal
   Enabled: true
-  EnforcedStyle: double_quotes
+  EnforcedStyle: single_quotes
```

Comment:
The performance in Ruby is the same single v's double quotes.  This only allows double quotes when interpolation is needed.  It's more about the semantics of strings etc than anything else.